tests/euler_aeos/verification-rarefaction-pge-1d-erk33-l6.prm

```diff
@@ -48,6 +48,10 @@ end
 subsection F - HyperbolicModule
   set cfl with boundary dofs        = true
 
+  subsection indicator
+    set indicator strategy = galerkin
+  end
```

Comment:
@ejtovar It is slightly annoying that we now have to set the indicator to `galerkin` for EulerAEOS (instead being controlled by the `strict_bounds` boolean). But in light of us wanting to fix the indicator (and improve EulerAEOS) anyway I suggest to keep it this way for now.